### DISCUSSION TITLE
AddressBook v1.1

### DISCUSSION THREAD

**Discussion by aljawaid** [aljawaid/AddressBook#5] [Announcements]
## v1.1

_(most recent changes are listed on top):_
- 527c364 FIX: Edit Modal Code Syntax
- 93dc636 FIX: Missing Visbility for Method
- 04c6e30 FIX: Task Contacts Table Tooltip Titles in Board View - only shown when hovering the text, not the full cell
- 38a3c98 FIX: Available Contacts Table Tooltip Titles - only shown when hovering the text, not the full cell
- b17118d FIX: Linked Contacts Table Tooltip Titles - only shown when hovering the text, not the full cell
- 5b25cf3 FIX: Task Contacts Table Tooltip Titles - only shown when hovering the text, not the full cell
- 3b9e926 FIX: Project Contacts Table Tooltip Titles - only shown when hovering the text, not the full cell
- 035118d FIX: Padding & Margins for Buttons
- 1138eb5 FIX: View Contact Button Consistency
- dd9b793 FIX: Task Contacts Table Not Showing Values in Board View Tooltip - this should fix the problem of adding/removing properties/sets which resulted in the property id not being in sync
- 62aa02e FIX: Task Contacts Table Not Showing Values in Task Summary - this should fix the problem of adding/removing properties/sets which resulted in the property id not being in sync
- c795790 FIX: Available Contacts Table Not Showing Values in Task Contacts - this should fix the problem of adding/removing properties/sets which resulted in the property id not being in sync
- bce852c FIX: Linked Contact Table Not Showing Values in Task Contacts - this should fix the problem of adding/removing properties/sets which resulted in the property id not being in sync
- 716a89d FIX: Contact Tables Not Showing Values in Project Contacts - this should fix the problem of adding/removing properties/sets which resulted in the property id not being in sync
- fa345ba FIX: Contact ID in Contact Edit was Preventing Form Submission - fixes 5ae8989
- 37341e8 FIX: Close Modal Icon Alignment for Contact View
- 1aebd07 FIX: Contact Routes
- 0b4de2e FIX: Add Project Address Book Routes
- 72609f3 FIX: Add Property Set Routes
- 4f41643 FIX: URL Route - changed `rename` to `edit`
- 731974c FIX: Add `#` `°` Symbols to Variable
- 4f201e8 FIX: KanboardCSS Compatibility
- a2df863 FIX: Property Set Button Margin
- 926b5bc FIX: Contact Profile Table Column Widths
- 4826587 FIX: Edit Contact Modal Sizing - for less than 3 fields
- 6e76b45 FIX: Close Modal Icon Alignment
- 4e4a0e4 FIX: Styling for Required Fields
- c0466dc FIX: Page Titles' Spacing
- d95f34c FIX: Translations - better wording
- 5eb3111 FIX: Plugin Failing to Load Redundant JS File

---

Read the full [**Changelog**](../master/changelog.md "See changes") or view the [**README**](../master/README.md "View README")

<hr /><em>This discussion was created from the release <a href='https://github.com/aljawaid/AddressBook/releases/tag/v1.1'>AddressBook v1.1</a>.</em>

**Comment by PopovIG** [comment]
Hello! =)
Thanks for your work, I think this plugin can be very useful.
But I have a problem - the plugin does not start.
I tried to install it manually (Download the .zip file and decompress everything under the directory /plugins) and automatically (Plugin Directory), the result is the same - the plugin does not even appear in the Plugins section.
I noticed that if you rename / delete the Scheme directory, the plugin starts to be displayed, but of course it does not work.

Application version: 1.2.22

**Reply by aljawaid** [reply]
Hello.... 

Please provide more information:

- What version ar you trying?  Stable version is v1.1
- What database are you using?  It is fully tested with MySQL
- > I noticed that if you rename / delete the Scheme directory, the plugin starts to be displayed, but of course it does not work.
  - This indicates something to do with the database, most likely the schema file for your database type might need looking into on my part but I need to know what type you use.  
  - Please check your error logs and see if you get any errors

**Reply by PopovIG** [reply]
- v1.1
- Sqlite

Found in debug.log: 
[2023-07-03 09:56:01] [critical] AddressBook: Unable to migrate schema for the plugin: AddressBook => SQLSTATE[HY000]: General error: 1 near "AUTO_INCREMENT": syntax error

**Reply by aljawaid** [reply]
Excellent, I will look into it shortly

**Reply by aljawaid** [reply]
@PopovIG This should be fixed in v1.2... thanks for reporting it to me and please star the project to show your support
